app/src/main/java/org/simple/clinic/widgets/qrcodescanner/OnQrCodeDetected.kt

```diff
@@ -1,3 +0,0 @@
-package org.simple.clinic.widgets.qrcodescanner
-
-typealias OnQrCodeDetected = (String) -> Unit
```

Comment:
Did we delete this file and created the same in a different package? Want to understand why the typealias is red in color if we're using the same thing.